docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,137 @@
+# Integration with an AWS Kubernetes cluster
+This guide will walk you through how to use Calico Networking with an existing AWS-Kubernetes cluster.
+
+## Requirements
+* A working Kubernetes Deployment on AWS with one master and at least one minion
+    - While any AWS deployment will do, we recommend the Kubernetes [`kube-up` guide](https://github.com/kubernetes/kubernetes/blob/release-1.0/docs/getting-started-guides/aws.md) for AWS. This guide was created with the `kube-up` script in mind.
```

Comment:
I think this can be made a lot more concise.  Instead of "While any AWS deployment will do...", replace that bullet with "We recommend the following set of instructions which are provided with Kubernetes" or the like.